### DISCUSSION TITLE
possible to have forecast grapth: precipication of rain and snow at the same time?

### DISCUSSION THREAD

**Discussion by robin1741** [FelixdelasPozas/TrayWeather#104] [Ideas]
Hi, im new to github. so idk if this is the right place to type this but. as the titel says. right now in sweden we have both rain and snow. so i have a future request to add. you can have both rain and snow at the same time???/Best regards Robin

**Comment by FelixdelasPozas** [comment]
Hi @robin1741 .

Yes, its possible. The reason there's only two variables in the forecast graph its the measurements. In the left side of the graph is the temperature and the right is the precipitation, but given that snow and rain have the same measurement (precipitation in mm or inches) both could be present at the same time. I'll look into it in the following days. 

![forecast](https://user-images.githubusercontent.com/12167134/162380231-fee8b7a3-0d4f-44d8-b42a-98aa71f291ac.png)

Regards, 
Félix.

**Comment by FelixdelasPozas** [comment]
Hi @robin1741. 

For the next release (1.21) you'll be able to customize a little the forecast graph. In the configuration options you can choose the representation for the values in the graph (bars or a smooth line) and also its color.
![Clipboard01](https://user-images.githubusercontent.com/12167134/163574798-e8b354d3-ccf2-4ae4-b208-5375fc1cb13a.png)

Then in the graphs there is the option to toggle the visualization of data by clicking in the legend text. 
![Clipboard02](https://user-images.githubusercontent.com/12167134/163574840-c5e2bb10-04fb-4e2d-a6f7-a94c72238ec1.png)
![Clipboard03](https://user-images.githubusercontent.com/12167134/163574843-ae7836b3-64cf-4b58-ae84-696f862714a3.png)

I'll hope this will solve your problem with the forecast graph. 

Kind regards, 
Félix.

**Reply by robin1741** [reply]
nice!!! can't wait to try it out!!!. thanks alot for adding/fixing this, much love. i really like this program :)/Best regards Robin

**Comment by FelixdelasPozas** [comment]
Hi @robin1741 . Version 1.21 with the changes [has been released](https://github.com/FelixdelasPozas/TrayWeather/releases/tag/1.21.0).

Regards,
Félix.
